galleries/examples/text_labels_and_annotations/demo_annotation_box.py

```diff
@@ -26,46 +34,37 @@
 # Annotate the 1st position with a text box ('Test 1')
 offsetbox = TextArea("Test 1")
 
-ab = AnnotationBbox(offsetbox, xy,
-                    xybox=(-20, 40),
-                    xycoords='data',
-                    boxcoords="offset points",
-                    arrowprops=dict(arrowstyle="->"),
-                    bboxprops=dict(boxstyle="sawtooth"))
-ax.add_artist(ab)
+ab1 = AnnotationBbox(offsetbox, xy,
+                     xybox=(-20, 40),
+                     xycoords='data',
+                     boxcoords="offset points",
+                     arrowprops=dict(arrowstyle="->"),
+                     bboxprops=dict(boxstyle="sawtooth"))
+ax.add_artist(ab1)
 
 # Annotate the 1st position with another text box ('Test')
-offsetbox = TextArea("Test")
-
-ab = AnnotationBbox(offsetbox, xy,
-                    xybox=(1.02, xy[1]),
-                    xycoords='data',
-                    boxcoords=("axes fraction", "data"),
-                    box_alignment=(0., 0.5),
-                    arrowprops=dict(arrowstyle="->"))
-ax.add_artist(ab)
+offsetbox = TextArea("Test 2")
 
-# Define a 2nd position to annotate (don't display with a marker this time)
-xy = [0.3, 0.55]
+ab2 = AnnotationBbox(offsetbox, (1, .85),
+                     xybox=(.75, xy[1]),
+                     xycoords=ab1,
+                     boxcoords=("axes fraction", "data"),
+                     box_alignment=(0., 0.5),
+                     arrowprops=dict(arrowstyle="->"))
+ax.add_artist(ab2)
 
-# Annotate the 2nd position with a circle patch
-da = DrawingArea(20, 20, 0, 0)
-p = Circle((10, 10), 10)
-da.add_artist(p)
+# %%%%
+# Images
+# ======
+# The `.OffsetImage` container supports plotting images using `.BboxImage`
```

Comment:
Mentioning `.BboxImage` does not add value here. It's not used in the code and I assume most users don't know it either.

I suggest to leave this out. Possibly the better place for that information is the `.OffsetImage` docstring.

Reply:
I believe the primary problem is that `OffsetImage` has no docstrings. That's bad in itself. And it makes the link earlier in this line pretty useless. The fix is to document `OffsetImage` so that it's easy to understand, not to try and compensate the missing docs here.